### DISCUSSION TITLE
数据放在代码中，在前端页面显式时不以代码块的方式显式么？

### DISCUSSION THREAD

**Discussion by duguyue8** [halo-dev/halo#4405] [Q&A]
下面的这段数据在编辑文章时是放在代码块中的，在前端页面展示时，以普通的文字展示的，不好看。
![代码块中的文字在前端显式](https://pic7.58cdn.com.cn/nowater/webim/big/n_v203b4fa05b7e443d8b27e02aa8dbdb2b9.png)

**Comment by JohnNiang** [comment]
尝试安装代码高亮插件呢：<https://halo.run/store/apps/app-sqpgf>。

**Reply by duguyue8** [reply]
多谢，已实现了代码块高亮的效果
![效果](https://pic7.58cdn.com.cn/nowater/webim/big/n_v2ccc9ccda60de45d7b3e6bc18aba982f7.png)
有类似于其它网站，代码块右上角一键复制的功能么？

**Reply by JohnNiang** [reply]
或许可以在这里提交 Issue：<https://github.com/halo-sigs/plugin-highlightjs/issues>。

**Reply by duguyue8** [reply]
very good!Submitted
